### ISSUE TITLE
Issue with jquery after closing the player

### ISSUE THREAD

**Issue by kalwalt** [augmentmy-world/arStudio#21]
This is the error:

```js
jquery.min.js:2 Uncaught Error: Syntax error, unrecognized expression: @ENC
    at Function.se.error (jquery.min.js:2)
    at se.tokenize (jquery.min.js:2)
    at se.select (jquery.min.js:2)
    at Function.se [as find] (jquery.min.js:2)
    at k.fn.init.find (jquery.min.js:2)
    at new k.fn.init (jquery.min.js:2)
    at k (jquery.min.js:2)
    at ArControllerComponent.stopAR (arcontroller.compone…js?version=0.64:321)
    at Object.trigger (litegl.js?version=0.64:10617)
    at Scene.finish (litescene.js?version=0.64:14467)
```

**Comment by kalwalt** [comment]
Maybe it is caused by a change in jquery syntax. I need to verify this. i can test with an older jquery version.

**Comment by kalwalt** [comment]
> Maybe it is caused by a change in jquery syntax. I need to verify this. i can test with an older jquery version.

no it not depends by jquery version. i have just tested with the old one.

**Comment by kalwalt** [comment]
At line 176 of **arcontroller.component.js** it should be:

 ```diff
+ var selectedCanvas = this.canvas[0];
- var selectedCanvas = $(this.canvas[0]);
```
I will fix tomorrow!

**Comment by kalwalt** [comment]
The issue maybe was caused because the `this.canvas=$('canvas');` was part of the arcomponent removing it, and changed the ref. code, solved the issue.

**Comment by kalwalt** [comment]
Solved with #19
